schemas/database/001_initial.sql

```diff
@@ -8,3 +8,23 @@ CREATE TABLE transactions (
     raw         BYTEA                    NOT NULL,
     hash        BYTEA                        NULL
 );
+
+CREATE TABLE log_fetches
+(
+    id          SERIAL PRIMARY KEY NOT NULL,
+    timestamp   TIMESTAMP          NOT NULL,
+    first_block BIGINT             NOT NULL,
+    last_block  BIGINT             NOT NULL,
+);
+
+CREATE TABLE logs
+(
+    block_index         BIGINT         NOT NULL,
+    transaction_index   BIGINT         NOT NULL,
+    log_index           BIGINT         NOT NULL,
+    fetch_id            INT            NOT NULL REFERENCES log_fetches (id),
```

Comment:
We don't need this, beacuse 1) I don't see a case for ever wanting to programatically know how we fetched the data. That level of verbosity belongs in logs. 2) We can query `log_fetches` for the block number.